### ISSUE TITLE
[Claims Details V2]: Enhance show past updates button so its state is understood by assistive technology

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#78904]
## Value Statement
**_As a_** Veteran who uses a screen reader or assistive techlogy
**_I want to_** understand the state of buttons on the page
**_So that_** I can understand what's happening on the page.

---
### Bug

**Issue description**

URL: `/track-claims/your-claims/600351984/overview`

The show/hide past updates buttons expand like accordions, but don't include any ARIA to communicate state, like an accordion would. This causes issues, especially for screen reader users because they won't know the state of the button.

<img width="684" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN2Y1QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3c3a8613a7c1c2ee94ab6befc026cc3dd60dd4e5/Screen%20Shot%202024-03-20%20at%206.14.44%20PM.png" alt="Overview of Claims Process page with two arrows pointing to the past updates buttons." />

This is an [a11y-defect-1](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)

**Remediation guidance**

Add the proper ARIA, like the [accordion component](https://design.va.gov/components/accordion) or use that instead. It needs two key ARIA attributes:

```html
<button type="button" class="usa-accordion__button" aria-expanded="true" aria-controls="content"><span class="usa-accordion__header va-accordion__header"><slot name="icon"></slot>First Amendment</span></button>
```

`aria-controls` points to the container being expanded.

**Does this impact visual design?**

Potentially, depending on the decision made.

Relevant standards

[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] New ARIA attributes are added.

**Comment by skylerschain** [comment]
I agree that adding the ARIA label is the best and quickest path forward for this.
